tests/manage/pv_services/test_pvc_concurrent_deletion_creation.py

```diff
@@ -22,134 +22,72 @@ def test_fixture(request):
     """
     Setup and teardown
     """
-    self = request.node.cls
+    cls_ref = request.node.cls
+    cls_ref.pvc_objs_new = []
 
     def finalizer():
-        teardown(self)
-    request.addfinalizer(finalizer)
-    setup(self)
-
-
-def setup(self):
-    """
-    Create new project
-    Create PVCs
-    """
-    # Create new project
-    self.namespace = create_unique_resource_name('test', 'namespace')
-    self.project_obj = ocp.OCP(kind='Project', namespace=self.namespace)
-    assert self.project_obj.new_project(self.namespace), (
-        f'Failed to create new project {self.namespace}'
-    )
-    # Create 100 PVCs
-    pvc_objs = create_multiple_pvcs(
-        sc_name=self.sc_obj.name, namespace=self.namespace,
-        number_of_pvc=self.number_of_pvc
-    )
-    log.info(f'Created initial {self.number_of_pvc} PVCs')
-    self.pvc_objs_initial = pvc_objs[:]
-
-    # Verify PVCs are Bound and fetch PV names
-    for pvc in self.pvc_objs_initial:
-        pvc.reload()
-        assert pvc.status == constants.STATUS_BOUND, (
-            f'PVC {pvc.name} is not Bound'
-        )
-        self.initial_pvs.append(pvc.backed_pv)
-    log.info(f'Initial {self.number_of_pvc} PVCs are in Bound state')
-
-
-def teardown(self):
-    """
-    Delete PVCs
-    Delete project
-    """
-    # Delete newly created PVCs
-    assert delete_pvcs(self.pvc_objs_new), 'Failed to delete PVCs'
-    log.info(f'Newly created {self.number_of_pvc} PVCs are now deleted.')
-
-    # Switch to default project
-    ret = ocp.switch_to_default_rook_cluster_project()
-    assert ret, 'Failed to switch to default rook cluster project'
+        # Delete newly created PVCs
+        assert delete_pvcs(cls_ref.pvc_objs_new), 'Failed to delete PVCs'
```

Comment:
@RazTamir @jilju  should we change the usage here to the new helpers.delete_pvcs() added in https://github.com/red-hat-storage/ocs-ci/pull/476 after it's merged?

Reply:
Let's merge this one first and I will take care of this in my PR